Rename utility `isClient` to `isBrowser` (#1389)
Shopify/hydrogen: docs/utilities/isbrowser.md

---
gid: 232fa7f6-5c17-4659-b8c8-4466be88d602
title: isBrowser

> description: The isBrowser utility is a function that returns a boolean indicating if the code was run on the client.

Reviewer: ```suggestion
description: The isBrowser utility is a function that returns a Boolean indicating if the code was run in the browser.
```

---

> The `isBrowser` utility is a function that returns a boolean indicating if the code was run in the browser. This is useful because client components are still server rendered. Use the `isBrowser` hook if you don't want logic within your client component to be rendered on the server.

Reviewer: ```suggestion
The `isBrowser` utility is a function that returns a Boolean indicating if the code was run in the browser. This is useful because client components are server-rendered. Use the `isBrowser` utility if you don't want logic within your client component to be rendered on the server.
```

## Example code

```tsx
import {isBrowser} from '@shopify/hydrogen';

export function MyComponent() {
  if (isBrowser()) {

>     return <p>I ran on the client</p>;

Reviewer: ```suggestion
    return <p>I ran in the browser</p>;
```

  }

  return <p>I ran on the server</p>;
}
```

## Arguments

None

## Return type

> A Boolean indicating if the code was run on the client.

Reviewer: ```suggestion
A Boolean indicating if the code was run in the browser.
```

## Related utilities

> - [`isServer`](api/hydrogen/utilities/isserver)

Reviewer: ```suggestion
- [`isServer`](https://shopify.dev/api/hydrogen/utilities/isserver)
```
